htdocs/js/contextualhover.js

```diff
@@ -329,18 +329,23 @@ ContextualPopup.renderPopup = function (ctxPopupId) {
         // member of community
         if (data.is_logged_in && data.is_comm) {
             var membership      = document.createElement("span");
+            var membership_action;
 
             if (!data.is_closed_membership || data.is_member) {
                 var membershipLink  = document.createElement("a");
 
-                var membership_action = data.is_member ? "leave" : "join";
```

Comment:
Let's put the `var membership_action` statement back here where it was previously, just to keep the scope the same.